test/integration/multiplexed_upstream_integration_test.cc

```diff
@@ -269,6 +269,35 @@ TEST_P(Http2UpstreamIntegrationTest, ManySimultaneousRequestsTightUpstreamLimits
   manySimultaneousRequests(1024, 1024, 10);
 }
 
+TEST_P(Http2UpstreamIntegrationTest, ManySimultaneousRequestsLaxUpstreamLimits) {
+  if (upstreamProtocol() == Http::CodecType::HTTP2) {
```

Comment:
Does this imply that this test will run for HTTP/1 and HTTP/3? Or is this only for HTTP/3 (since the file is multiplexed_upstream and HTTP/1 is not multiplexed?). If it's only HTTP/3, then maybe:
```
if (upstreamProtocol() != Http::CodecType::HTTP3) {
  return;
}
```
(and skip the conditional at line 283).

Alternatively, if it runs for HTTP/1, maybe add a comment?

Reply:
was cargo culted from prior test - it actually works for both so updating.